src/show/pages/public/CreateProject/Subviews/index.js

```diff
@@ -0,0 +1,443 @@
+import ReactDatePicker from 'react-datepicker';
+import { categories } from '../../../../../process/constants';
+import { BsArrowBarLeft, BsArrowBarRight } from 'react-icons/bs';
+import { Button, ProgressBar, TextField } from '../../../../components';
+
+import 'react-datepicker/dist/react-datepicker.css';
+
+const CategoryStep = ({ nextStep, project, setProject }) => {
+  return (
+    <div>
+      <div className='mt-36 flex flex-col text-center text-3xl text-primary-600 '>
+        Choose a category
+      </div>
+      <div className='mt-10 flex justify-evenly h-32'>
+        {categories.map((category, index) => (
+          <div
+            onClick={() => {
+              nextStep();
+              setProject({
+                ...project,
+                category: category.name
+              });
+            }}
+            className='cursor-pointer hover:scale-110 transition-all duration-300'
+            key={index}>
+            <div className='flex justify-evenly bg-white mt-5 text-primary-600'>
+              {category.icon}
+            </div>
+            <div className='text-center mt-2 text-gray-500'>
+              {category.name}
+            </div>
+          </div>
+        ))}
+      </div>
+      <div className='mt-72 px-80'>
+        <ProgressBar percentage={10} color='bg-primary-600' />
+      </div>
+    </div>
+  );
+};
+
+const TitleStep = ({ nextStep, previousStep, project, setProject }) => {
+  return (
+    <div className='mt-36 px-20'>
+      <div className='text-center text-3xl text-gray-500'>
+        Name your project
+      </div>
+      <div className='h-64'>
+        <TextField
+          required
+          value={project.name}
+          className='rounded-2xl mt-2 items-center text-center'
+          onChange={e => setProject({ ...project, name: e.target.value })}
+        />
+      </div>
+      <div className='flex mt-20 justify-between cursor-pointer'>
+        <div
+          onClick={() => previousStep()}
+          className='flex items-center hover:scale-110 transition-all duration-300'>
+          <BsArrowBarLeft size='2rem' />
+          Back
+        </div>
+        <div
+          onClick={() => nextStep()}
```

Comment:
Can I still click next if didn't type in any name for my project? 😇 

^ applies to every step.